sat/cli/bootsys/bos.py

```diff
@@ -253,12 +263,11 @@ def has_completed(self):
             if session_status['error_summary']:
                 self.error_summary = session_status['error_summary']
 
-            if session_status['status'] == self.target_state:
+            if session_status['status'] in self.target_states:
```

Comment:
There was a conflict here due to a change in integration that we haven't backported to release/3.21. Can you double-check that I handled it correctly, @jack-stanek-hpe?